src/components/NationalText/utils.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { Fragment } from 'react';
+import AggregationsJSON from 'assets/data/aggregations.json';
+import { last, sum, isNumber } from 'lodash';
+import moment from 'moment';
+import {
+  formatPercent,
+  formatDecimal,
+  getPercentChange,
+  formatEstimate,
+} from 'common/utils';
+
+const getTwoWeekPercentChange = (series: any[]): number | null => {
+  const lastTwoWeeks = series.slice(-14);
+  const firstVal = last(lastTwoWeeks);
+  const lastVal = lastTwoWeeks[0];
+  if (!firstVal || !lastVal) {
+    return null;
+  }
+  return getPercentChange(firstVal, lastVal);
+};
+
+const getTotalCasesCopy = (summedRawCases: number): string => {
+  const ONE_MILLION = 1000000;
+  const divided = summedRawCases / ONE_MILLION;
+  return `${formatDecimal(divided, 1)} million`;
+};
+
+export const getNationalText: React.FC = () => {
+  const usaAggregation = AggregationsJSON['00001'];
+  const {
+    rawDailyCases,
+    rawDailyDeaths,
+    smoothedDailyCases,
+    smoothedDailyDeaths,
+    dates,
+  } = usaAggregation;
+
+  const totalCases: number = sum(rawDailyCases);
+
+  const totalDeaths: number = sum(rawDailyDeaths);
+
+  const percentChangeSmoothedCases = getTwoWeekPercentChange(
+    smoothedDailyCases,
+  );
+  const percentChangeSmoothedDeaths = getTwoWeekPercentChange(
+    smoothedDailyDeaths,
+  );
+
+  const renderPercentChangeSentence =
+    isNumber(percentChangeSmoothedCases) &&
+    isNumber(percentChangeSmoothedDeaths);
+
+  const lastDate = last(dates);
+  const lastDateFormatted: string = moment(lastDate).format('MMMM Do, YYYY');
+
+  const getChangeDescriptorCopy = (percentChange: number): string => {
+    const changeByCopy = `by about ${formatPercent(Math.abs(percentChange))}`;
+    if (percentChange > 0.01) {
+      return `increased ${changeByCopy}`;
+    } else if (percentChange < 0) {
+      return `decreased ${changeByCopy}`;
+    } else return 'remained about consistent each day';
+  };
+
+  return (
+    <Fragment>
+      As of {lastDateFormatted}, there have been roughly{' '}
+      {getTotalCasesCopy(totalCases)} cases and{' '}
+      {formatEstimate(totalDeaths, 3).toLocaleString()} deaths from COVID in the
+      United States.{' '}
+      {renderPercentChangeSentence
+        ? `Over the last 14 days, daily new
+    cases have ${getChangeDescriptorCopy(
+      percentChangeSmoothedCases!,
```

Comment:
even with the `renderPercentChangeSentence` check, this still yells at me when theres not a `!`. same with `percentChangeSmoothedDeaths!` below

Reply:
That's ok, we can keep the `!` 🤷

Reply:
FWIW- TypeScript has no way to know that if the lodash `isFinite()` function returns true then the value you passed wasn't null.  If you wanted to get rid of the `!` here, you could do explicit null checks:

```
  const renderPercentChangeSentence =
    percentChangeSmoothedCases !== null &&
    percentChangeSmoothedDeaths !== null;
```

And then TypeScript wouldn't complain that it might be null.  

Feel free to leave as-is though.